storage/migrations/16_validator_staking_history.up.sql

```diff
@@ -0,0 +1,16 @@
+BEGIN;
+
+CREATE TABLE chain.validator_balance_history
```

Comment:
```suggestion
CREATE TABLE chain.validator_epoch_history
```

Maybe it makes it clearer that this is per-epoch history and we might have non-balance data in here in future as well (such as number of delegators etc might be useful).

And IMO similarly also rename the validator balances analyzer.

Reply:
Went with `history.validators`, it seemed to be the most accurate/concise.